Dockerfile

```
@@ -58,8 +58,8 @@ RUN set -ex && cd ~ \
   && mv terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64 /usr/local/bin/terraform-docs
 
 # install circleci cli
-ARG CIRCLECI_CLI_VERSION=0.1.5879
-ARG CIRCLECI_CLI_SHA256SUM=f178ea62c781aec06267017404f87983c87f171fd0e66ef3737916246ae66dd6
+ARG CIRCLECI_CLI_VERSION=0.1.6772
```

Any reason why not to make it the one they released 10 hrs ago?
```suggestion
ARG CIRCLECI_CLI_VERSION=0.1.6893
```

---

No reason, just picked the one that upgraded when I upgrade homebrew this morning. Homebrew is probably behind.